starknet-rpc-test/spec_version.rs

```diff
@@ -1,15 +1,13 @@
 extern crate starknet_rpc_test;
 
 use rstest::rstest;
-use starknet_providers::Provider;
-use starknet_rpc_test::constants::SPEC_VERSION;
 use starknet_rpc_test::fixtures::{madara, ThreadSafeMadaraClient};
 
 #[rstest]
 #[tokio::test]
 #[ignore = "Waiting for starknet_providers::jsonrpc upgrade to v0.6.0"]
 async fn returns_hardcoded_spec_version(madara: &ThreadSafeMadaraClient) -> Result<(), anyhow::Error> {
-    let rpc = madara.get_starknet_client().await;
+    madara.get_starknet_client().await;
```

Comment:
You can entrierly remove that line?

Reply:
I just removed this because of `clippy` complaining, the test is not ran anyway for now